php/blocks/migration/class-post-content.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Post_Content.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2020, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Migration;
+
+use WP_Error;
+
+/**
+ * Class Post_Content
+ */
+class Post_Content {
+
+	/**
+	 * The previous namespace of the block.
+	 *
+	 * @var string
+	 */
+	private $previous_block_namespace;
+
+	/**
+	 * The new namespace of the block.
+	 *
+	 * @var string
+	 */
+	private $new_block_namespace;
+
+	/**
+	 * Post_Content constructor.
+	 *
+	 * @param string $previous_block_namespace Previous namespace of the blocks.
+	 * @param string $new_block_namespace      New namespace of the blocks.
+	 */
+	public function __construct( $previous_block_namespace, $new_block_namespace ) {
+		$this->previous_block_namespace = $previous_block_namespace;
+		$this->new_block_namespace      = $new_block_namespace;
+	}
+
+	/**
+	 * Migrates all of the block namespaces in all of the posts that have Block Lab blocks.
+	 *
+	 * @return array The results of migration, either int or WP_Error.
+	 */
+	public function migrate_all() {
+		$results            = [];
+		$error_count        = 0;
+		$max_allowed_errors = 20;
+		$posts              = $this->query_for_posts();
+
+		while ( $posts && $error_count < $max_allowed_errors ) {
+			foreach ( $posts as $post ) {
+				if ( isset( $post->ID ) ) {
+					$migrated_post = $this->migrate_single( $post->ID );
+					$results[]     = $migrated_post;
```

Comment:
Before releasing this, we might remove `$results`, and only return `WP_Error` messages if there were any. If there are hundreds of posts migrated, this could be too big.

But for now, it lets us test this and get the WP-CLI output of exactly which posts were migrated.